DataFormats/Common/interface/DetSetVectorNew.h

```diff
@@ -30,20 +40,41 @@ namespace edmNew {
 
   namespace dslv {
     template< typename T> class LazyGetter;
+
   }
 
   /* transient component of DetSetVector
-   * for pure conviniency of dictioanary declaration
+   * for pure conviniency of dictionary declaration
    */
   namespace dstvdetails {
+
+    void errorFilling();
+    void notSafe();
+    void errorIdExists(det_id_type iid);
+    void throw_range(det_id_type iid);
+
     struct DetSetVectorTrans {
       DetSetVectorTrans(): filling(false){}
-      bool filling;
+#ifndef USE_ATOMIC
```

Comment:
Given this is not a storable class, I believe ROOT 6 is fine with seeing this ```std::atomic```. If that is the case than this conditional compilation guard is not needed.